src/helpers/validator.js

```diff
@@ -0,0 +1,50 @@
+import { check } from 'express-validator';
+
+/**
+ *  A class that provide static method to validate user input
+ */
+class Validator {
```

Comment:
why are we using classes?

Reply:
I use a class because I want to encapsulate the code, the class is like a wrapper around all the methods in it. using a class, I can easily export the class and use all its method in one place.

Reply:
I'm of the opinion that exporting functions here is a better option as we don't have any class properties we're using and won't be extending this class. At the end of the day, writing classes still falls back to `function ClassName(){}` and `ClassName.prototype.classMethods = function(){}`

Reply:
ok